src/dnsimple/Services/Registrar.cs

```diff
@@ -226,8 +261,7 @@ public struct DomainTransfer
         public string State { get; set; }
         public bool AutoRenew { get; set; }
         public bool WhoisPrivacy { get; set; }
-
-        public string AuthCode { get; set; }
```

Comment:
auth code is only needed in DomainTransferInput, it is not answered by the API.

Reply:
If you set the property `ItemNullValueHandling = NullValueHandling.Ignore` for the struct then the property will not be rendered in the JSON when they are `null` (the AuthCode property would not even appear in the JSON). It might be a different way of handling this instead of removing the property.

Reply:
@ecomba DomainTransfer doesn't have that attribute at all in its definition. It is probably a good idea to remove it:

https://github.com/dnsimple/dnsimple-developer/blob/f04a9f3982c7b1d9206cb1012e21257143ea0f4d/content/v2/openapi.yml#L2889-L2925